cohort-evaluator-spark/src/main/java/com/ibm/cohort/cql/terminology/R4FileSystemFhirTerminologyProvider.java

```diff
@@ -210,8 +197,26 @@ public boolean accept(Path path) {
 								.parseResource(new InputStreamReader(fileSystem.open(valueSetFiles[0].getPath())));
 						LOG.info("Unmarshalled json {}", valueSetFhirR4.getId());
 					}
+					
+					//Add the codes to a HashSet using a wrapper object with the correct hashCode/equals behavior
```

Comment:
Your comment here is a little out of date now since you are no longer using the custom Code object with hashCode/equals logic.

Reply:
fixed